operator/rust/crates/operator/src/test_utils.rs

```diff
@@ -285,25 +328,40 @@ async fn start_creating_tasks() {
 mod tests {
     use super::*;
     use dotenv::dotenv;
-    use eigen_utils::binding::DelegationManager::{self, isOperatorReturn};
+    use eigen_logging::init_logger;
+    use eigen_utils::delegationmanager::DelegationManager::{self, isOperatorReturn};
+    use serial_test::serial;
+    use std::path::Path;
     use HelloWorldServiceManager::latestTaskNumReturn;
     #[tokio::test]
+    #[serial]
     async fn test_register_operator() {
         dotenv().ok();
+        init_logger(eigen_logging::log_level::LogLevel::Info);
         let _ = register_operator().await;
 
-        let wallet = PrivateKeySigner::from_str(&KEY).expect("failed to generate wallet ");
-        let provider = get_signer(KEY.clone(), &RPC_URL);
-
-        let delegation_manager_contract_address =
-            Address::from_str(&DELEGATION_MANAGER_CONTRACT_ADDRESS)
-                .expect("wrong delegation manager contract address");
+        let signer = PrivateKeySigner::from_str(&KEY.clone()).unwrap();
+        let wallet = EthereumWallet::from(signer.clone());
+        let pr = ProviderBuilder::new()
+            .with_recommended_fillers()
+            .wallet(wallet)
+            .on_http(Url::from_str(&ANVIL_RPC_URL).unwrap());
+        let data = &format!("{}", env!("CARGO_MANIFEST_DIR"));
+        let mut path = Path::new(data);
+        for _ in 0..4 {
+            path = path
+                .parent()
+                .expect("Reached the filesystem root, no more parent directories");
+        }
```

Comment:
why do you need to do this?

Reply:
fixed. it was. not working in the test_utils.
it worked after moving to a separate get function in utils